.github/workflows/scanbuild.yml

```diff
@@ -48,10 +48,7 @@ jobs:
                 CC: ${{ matrix.compiler }}
                 MAKEVARS: ${{ matrix.makevars }}
               run: |
-                autoreconf -i
-                rm src/lexer.c src/lexer.h
-                rm src/parser.c src/parser.h
-                ./configure --with-oniguruma=builtin YACC="$(which bison) -y" $COVERAGE
+                ./configure --with-oniguruma=builtin $COVERAGE
```

Comment:
Was there a reason why scan-build would re-generate the lexer/parser code?

Reply:
I can't think of one.